pkg/operator/defaults.go

```diff
@@ -67,6 +67,10 @@ var (
 		"v2.23.0",
 		"v2.24.0",
 		"v2.24.1",
+		"v2.25.0",
+		"v2.25.1",
+		"v2.25.2",
+		"v2.26.0",
```

Comment:
v2.26.0 isn't released yet :)

Reply:
Well, the tag is already there so this should only be a matter of hours I hope :)
https://github.com/prometheus/prometheus/releases/tag/v2.26.0

Reply:
Now it is released :)

Reply:
:smile: relaunched the CI